exercises/resistor-color/test/test_resistor_color.c

```diff
@@ -1,7 +1,10 @@
+#include <assert.h>
 #include "vendor/unity.h"
 #include "../src/resistor_color.h"
 
 #define ARRAY_LENGTH(A) (sizeof(A) / sizeof(A[0]))
+_Static_assert(sizeof(resistor_band_t) == sizeof(int),
```

Comment:
Neat, apparently this is C11 and is actually standard. I've been stuck in C99 land and had no idea.

Reply:
The macro `static_assert` should also be available in C11, see [https://en.cppreference.com/w/c/error/static_assert](https://en.cppreference.com/w/c/error/static_assert). However, the `makefile` for the C track specifies `-std=c99`. Also, I'm not sure how MSVC handles this, they are pretty far behind on the C11 support I have heard.

Reply:
Very far behind, C99 support is not complete even. MS have stated there is no intention of catching up either.